src/components/Modal.scss

```diff
@@ -0,0 +1,20 @@
+@import 'variables.scss';
+
+.modal-container{
+    position: relative;
+    top:22vh;
+    right: 10vw;
```

Comment:
strange values: modal usually relies on fixed position. what are you trying to do?